lib/condor.py

```diff
@@ -260,7 +261,27 @@ def load_submit_file(filename: str) -> Tuple[Any, Optional[int]]:
     return htcondor.Submit(res), nqueue
 
 
-# pylint: disable=dangerous-default-value,too-many-locals,too-many-branches
+def ran_vault_storer_recently(schedd: str, handle: str, outbase: str) -> bool:
+    """See if we left a timestamp file from a previous run of jobsub_submit,
+    and it was within the last week minus a bit"""
+    try:
+        sv = os.stat(f"{outbase}/.cvs_{schedd}_{handle}")
+        # print(f"saw cvs file, sv.st_mtime is {sv.st_mtime} check: {sv.st_mtime > time.time() - 604500}")
+        # 1 week: 24 * 7 * 60 * 60 = 604800, les 5 minutes (300) -> 604500
+        return sv.st_mtime > time.time() - 604500
+    except FileNotFoundError:
+        return False
+
+
+def record_vault_storer_run(schedd: str, handle: str, outbase: str) -> None:
+    """touch a file to record when we last ran condor_vault_storer"""
+    # recreate/touch the file
+    with open(f"{outbase}/.cvs_{schedd}_{handle}", "w", encoding="utf-8"):
```

Comment:
I wonder if anyone would think that `.cvs` refers to the ancient code management system ... maybe `.stored` would avoid confusion.

Reply:
Agreed.  I actually was confused about that until I realized it stood for "condor_vault_storer". I'd even say ".stored_token....".

Reply:
Also, the more idiomatic way of doing this would be:

```python
pathlib.Path(f"{outbase}/.cvs_{schedd}_{handle}").touch(exists_ok=True)
```

That'd be a one-liner (which I still think should be in this function so it's clear to the reader WHY we're touching the file), and you wouldn't need the explanation - it'd be very clear.  Thoughts on using that instead?

Reference:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch

Reply:
Also done.